projects/ngrx.io/content/guide/effects/index.md

```diff
@@ -117,21 +122,19 @@ import { MoviesService } from './movies.service';
 
 @Injectable()
 export class MoviesEffects {
-
-  loadMovies$ = createEffect(() => this.actions$.pipe(
-    ofType('[Movies Page] Load Movies'),
-    exhaustMap(() => this.moviesService.getAll()
-      .pipe(
-        map(movies => ({ type: '[Movies API] Movies Loaded Success', payload: movies })),
-        catchError(() => EMPTY)
-      ))
-    )
-  );
-
-  constructor(
-    private actions$: Actions,
-    private moviesService: MoviesService
-  ) {}
+  private actions$ = inject(Actions);
+  private moviesService = inject(MoviesService);
+
+  loadMovies$ = createEffect(() => {
+    return this.actions$.pipe(
```

Comment:
the examples are also updates to be compliant to https://ngrx.io/guide/eslint-plugin/rules/prefer-effect-callback-in-block-statement